### PR TITLE
[TEVA-4214] only show statistics tab or tabs ui for published vacancies

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-4214

## After

Screenshot 2022-07-27 at 14 47 50

### CODE REVIEW

File: app/views/publishers/vacancies/_tabs_show.html.slim [hunk 1]
```diff
@@ -2,6 +2,7 @@
   - tabs.navigation_item text: t("buttons.manage_listing"), link: organisation_job_path(vacancy.id), active: controller_name == "vacancies"
   - if vacancy.can_receive_job_applications? && !current_organisation.local_authority?
     - tabs.navigation_item text: t("tabs.applications.hide_count"), link: organisation_job_job_applications_path(vacancy.id), active: controller_name == "job_applications"
-  - tabs.navigation_item text: t("tabs.statistics"), link: organisation_job_statistics_path(vacancy.id), active: controller_name == "statistics"
+  - if vacancy.listed?
```
Comment (cesidio) [inline app/views/publishers/vacancies/_tabs_show.html.slim:5]: `vacancy.listed?` will discard expired vacancies, for which you may still want to see the statistics maybe? if that's the case you want to use `vacancy.published?`

File: app/views/publishers/vacancies/show.html.slim [hunk 2]
```diff
@@ -4,7 +4,8 @@
 = vacancy_review(vacancy, step_process: step_process, back_to: "manage_draft", show_errors: params[:show_errors]) do |r|
   - r.header
     = render "banner", vacancy: vacancy, title: t("jobs.manage_listing", state: (vacancy.draft? ? "draft" : "job"))
-    = render "tabs_show", vacancy: vacancy
+    - if vacancy.listed?
```
Comment (cesidio) [inline app/views/publishers/vacancies/show.html.slim:7]: same here, I guess for expired vacancies you want to see this

File: app/views/vacancies/_show.html.slim [hunk 3]
```diff
@@ -7,7 +7,7 @@
 - content_for :breadcrumbs do
   = govuk_breadcrumbs breadcrumbs: vacancy_breadcrumbs(@vacancy)
 
-- if @vacancy.expired?
+- if @vacancy.published?
```
Comment (cesidio) [inline app/views/vacancies/_show.html.slim:10]: wrong place

Comment (alexbowen) [reply app/views/vacancies/_show.html.slim:10]: doh, im not doing well here

**Review by cesidio — APPROVED** [review]

**Pull request merged**